blocks/components/editable/index.js

```diff
@@ -271,14 +262,26 @@ export default class Editable extends wp.element.Component {
 		return !! this.state.formats[ format ];
 	}
 
-	toggleFormat( format ) {
+	changeFormats( formats ) {
 		this.editor.focus();
 
-		if ( this.isFormatActive( format ) ) {
-			this.editor.formatter.remove( format );
-		} else {
-			this.editor.formatter.apply( format );
-		}
+		Object.keys( formats ).forEach( format => {
```

Comment:
Could use Lodash's `forEach` which works on objects and would optimize from `O(2n)` to `O(n)`:

```js
forEach( formats, ( formatValue, format ) => {
```